sceptre/helpers.py

```diff
@@ -41,14 +45,14 @@ def recurse_into_sub_stack_groups(func):
     @wraps(func)
     def decorated(self, *args, **kwargs):
         function_name = func.__name__
-        responses = {}
+        responses = factory()
         num_stack_groups = len(self.sub_stack_groups)
-
+        print("sub stack groups: " + str(num_stack_groups))
```

Comment:
Wild print statement? Is it a debug one or should it be logged?

Reply:
Correct, completely wild. 😟 must have made it through an earlier fixup I did. Will get it removed.